target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -140,7 +140,7 @@ ramips_setup_interfaces()
 		ucidef_set_interface_lan "lan0 lan1 lan2 lan3"
 		;;
 	tplink,er605-v2)
-		ucidef_set_interfaces_lan_wan "eth1 eth2 eth3 eth4" "eth0"
+		ucidef_set_interfaces_lan_wan "lan1 lan2 lan3 lan4" "wan"
```

Comment:
This is the same as the default (` *) `) and you can drop this configuration to use the default.

Reply:
thanks! Updated

Reply:
As the device is labeled port 1-5, where port 1 is WAN, and the others can are labeled WAN/LAN 2-3 and LAN 4-5.

I would propose that we rename the LAN ports to LAN2-5 to make it consistent with the device labeling?

See: https://static.tp-link.com/upload/image-line/ER605_V2_VPN_Load_Balance_Router_01_large_20211125084350g.jpg

Reply:
TBH, I do not have a strong opinion about this, I can update the PR. I just would like to have all ports working :).
Would you prefer to also have `wan1`?

Reply:
Yes this is a great idea. 
`wan1` and `lan2-5` then for the ports would fit perfectly.